src/main/java/org/alfresco/opencmis/search/CMISResultSetRow.java

```diff
@@ -176,7 +177,11 @@ public Serializable getValue(String columnName)
         context.setScore(getScore());
         for (Column column : query.getColumns())
         {
-            if (column.getAlias().equals(columnName))
+            // When an SCORE selector is included, score must be adapted to range 0..1 due to CMIS specification
+            if (column.getFunction().getName().equals(Score.NAME)) {

```

Comment:
Braces should be on their own line.

Reply:
Could we have an NPE here?  E.g. If a column isn't a function?

Reply:
In fact, `function` can be Null:
https://github.com/Alfresco/alfresco-data-model/blob/master/src/main/java/org/alfresco/repo/search/impl/querymodel/impl/lucene/LuceneQueryModelFactory.java#L264

It is not Null for any of our 1,000 e-2-e tests, but there is a possibility to be null.